pkg/cmd/step/post/step_post_build.go

```diff
@@ -1,254 +0,0 @@
-package post
```

Comment:
I think at the very least this command is still used and should maybe not have been deprecated. See als0 https://github.com/jenkins-x-buildpacks/jenkins-x-kubernetes/blob/master/packs/go/pipeline.yaml#L12

Reply:
Added it back.